app/src/components/profile/public/ProfileRoles.tsx

```diff
@@ -3,6 +3,16 @@ import CheckIconRed from "@/components/icons/checkIconRed"
 import useAttestations from "@/hooks/api/useAttestations"
 import { UserWithAddresses } from "@/lib/types"
 
+const validAttestations = [
+  "Anticapture Commission",
+  "Citizen",
+  "Collective Feedback Commission",
+  "Developer Advisory Board",
+  "Grants Council",
+  "Security Council",
+  "Top 100 Delegate",
+]
```

Comment:
sorry, the ticket was a bit misleading -- all of the attestations are valid attestations. Let's just display `About` link on all of them